Matveibos/hometask-3/cli.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+ENTER_MESSAGE = 'Enter city name: '
+ERROR_MESSAGE = 'City name can not be blank!'
+EXIT_MESSAGE = 'exit!'
+
+# this class contains describes cli for rainfall stats
+class Cli
+  def run_cli
+    loop do
+      print ENTER_MESSAGE
+      input = gets.chomp
+
+      break if input == EXIT_MESSAGE
+
+      input.empty? ? (puts ERROR_MESSAGE) : rainfall_statistics(input)
```

Comment:
Lets pass `rainfall_statistics` result straight to the `show` method
```suggestion
      input.empty? ? (puts ERROR_MESSAGE) : show(rainfall_statistics(input))
```